### DISCUSSION TITLE
dependabot only look for action.yml when update github actions in workflow

### DISCUSSION THREAD

**Discussion by litaocdl** [community/community#35986] [Actions]
if my workflow file is not exists in .github/.workflows/ dir, i.e I configure my denpendabot to update github action versions like this

version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/root/.github/workflows/"
    schedule:
      interval: "daily"

it always looking for github actions in file /root/.github/workflows/action.yml , if I have action.yml exists, version will be update, if  did not have action.yml, non will be updated.

**Comment by antonblr** [comment]
There is an issue/feature request for this functionality - https://github.com/dependabot/dependabot-core/issues/4899
